examples/constraints.yml

```diff
@@ -16,60 +16,82 @@
           - attrs:
               - name: pcmk_host_list
                 value: node1 node2
-      - id: dummy-1
+      # wokeignore:rule=dummy
```

Comment:
do you need this wokeignore?  you have one below for the `ocf:pacemaker:Dummy`

Reply:
There are several other places below where it looks as though you have a wokeignore for the `dmy`?